src/ImageRunModal.jsx

```diff
@@ -0,0 +1,177 @@
+                    {_("Name")}
+                </label>
+                <input id='run-image-dialog-name'
+                    type='text'
+                    placeholder={_("Container Name")}
+                    value={dialogValues.containerName}
+                    onChange={e => this.onValueChanged('containerName', e.target.value)}
+                    className='form-control' />
+
+                <label className='control-label' htmlFor='run-image-dialog-command'>
+                    {_("Command")}
+                </label>
+                <input id='run-image-dialog-command'
+                    type='text'
+                    placeholder={_("Command")}
+                    value={dialogValues.command || ''}
+                    onChange={e => this.onValueChanged('command', e.target.value)}
+                    className='form-control' />
+
+                <label className='control-label' htmlFor='run-image-dialog-memory'>
+                    {_("Memory Limit")}
+                </label>
+                <div role='group' className='form-inline'>
+                    <div className="checkbox">
+                        <input id="run-image-dialog-memory-limit-checkbox" type="checkbox"
+                               checked={this.state.memoryConfigure}
+                               onChange={e => this.onValueChanged('memoryConfigure', e.target.checked)} />
+                    </div>
+                    <input className='form-control'
+                           type='number'
+                           value={dialogValues.memory}
+                           step={1}
+                           min={0}
+                           disabled={!this.state.memoryConfigure}
+                           onChange={e => this.onValueChanged('memory', e.target.value)} />
+                    <Select.Select id='memory-unit-select'
+                                   initial={this.state.memoryUnit}
+                                   enabled={this.state.memoryConfigure}
+                                   onChange={value => this.onValueChanged('memoryUnit', value)}>
+                        <Select.SelectEntry data={units.MiB.name} key={units.MiB.name}>
+                            {_("MiB")}
+                        </Select.SelectEntry>
+                        <Select.SelectEntry data={units.GiB.name} key={units.GiB.name}>
+                            {_("GiB")}
+                        </Select.SelectEntry>
+                    </Select.Select>
+                </div>
+
+                <label className='control-label'> {_("With terminal")} </label>
+                <label className="checkbox-inline">
+                    <input id="run-image-dialog-tty" type="checkbox"
+                           checked={this.state.hasTTY}
+                           onChange={e => this.onValueChanged('hasTTY', e.target.checked)} />
+                    {_("Allocate a pseudo-TTY")}
```

Comment:
I can't say I like this label much. It's very technical and not interesting (implementation detail, not an user-visible feature) and also redundant with the "With terminal" label.

This is not a veto though, we can change this later.

Reply:
Removed.